Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ocaml/opam
+
+RUN cd /home/opam/opam-repository && git pull
+
+RUN opam config exec -- opam update
+
+WORKDIR learn-ocaml
+
+RUN sudo chown -R opam:nogroup .
```

Comment:
@OCamlPro-Couderc just taking a look at the latest version of the Dockerfile, I was wondering whether it'd be better to move this command 3 lines below instead? (i.e., `RUN sudo chown -R opam:nogroup .` after `COPY Makefile Makefile`?)

Reply:
You're right, actually it is not even useful at that time of the build, and it is done once again once all the files have been into the container. I removed this `chown` in my next commits.